index/Dockerfile

```diff
@@ -25,6 +24,10 @@ RUN apt-get update \
     apt-get autoremove && \
     rm -rf /var/lib/{apt,dpkg,cache,log}
 
+# Install PostgreSQL Client 13
+RUN wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
```

Comment:
Can we do this line about the other apt installs please, and install postgres as part of that still.

Reply:
> Can we do this line about the other apt installs please, and install postgres as part of that still.

@alexgleith there are some dependancy packages needed (e.g. wget, lsb-release) to configure postgres repository and key so can't install it in the same place

Reply:
Ah ok... Then what we want to do is this:

```
    && apt-get autoclean \
    && apt-get autoremove \
    && rm -rf /var/lib/{apt,dpkg,cache,log}
```

after the apt commands. This keeps the image smaller.

Reply:
done